airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeCopyAzureBlobStorageDestination.java

```diff
@@ -63,6 +73,47 @@ public SqlOperations getSqlOperations() {
     return new SnowflakeSqlOperations();
   }
 
+  @Override
+  public AirbyteConnectionStatus check(final JsonNode config) {
+    try {
+      checkPersistence(config);
+    } catch (final Exception e) {
+      LOGGER.error("Exception attempting to access the staging persistence: ", e);
+      return new AirbyteConnectionStatus()
```

Comment:
There's a new [paradigm shift](https://github.com/airbytehq/airbyte/pull/18989/) where instead of returning failed `AirbyteConnectionStatus` within the `check` operations this will be caught within the `IntegrationRunner` class, so the new logic should be 

```java
catch (final Exception e) {
  LOGGER.error("Exception attempting to access the staging persistence: ", e);
  throw new ConfigErrorException(String.format("Could not connect to the staging persistence with the provided configuration. \n %s", e.getMessage());
}
```

Reply:
Removed this overridden method at all. Thanks